src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -155,12 +178,38 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
             if (response) {
                 const payload = {
                     data: response.data,
-                    indicatorId: indicatorId
+                    indicatorId
                 } as CalibrateResultWithType
                 commit({type: ModelCalibrateMutation.CalibrateResultFetched, payload: payload});
             }
             commit(`modelOutput/${ModelOutputMutation.SetTabLoading}`, {payload:{tab, loading: false}}, {root: true});
         }
+        updatePlotSelections(commit, payload);
+    }
+};
+
+const updatePlotSelections = (commit: Commit, payload: ResultDataPayload) => {
+    switch (payload.tab) {
+        case ModelOutputTabs.Bar:
+            commit(`plottingSelections/${PlottingSelectionsMutations.updateBarchartSelections}`, {
+                payload: payload.payload
+            }, {root: true});
+            break;
+        case ModelOutputTabs.Table:
+            commit(`plottingSelections/${PlottingSelectionsMutations.updateTableSelections}`, {
+                payload: payload.payload
+            }, {root: true});
+            break;
+        case ModelOutputTabs.Map:
+            commit(`plottingSelections/${PlottingSelectionsMutations.updateOutputChoroplethSelections}`, {
+                payload: payload.payload
+            }, {root: true});
+            break;
+        case ModelOutputTabs.Bubble:
+            commit(`plottingSelections/${PlottingSelectionsMutations.updateBubblePlotSelections}`, {
+                payload: payload.payload
+            }, {root: true})
+            break;
```

Comment:
Could shorten this a bit. Something like:
```suggestion
    let mutation: PlottingSelectionsMutations;
    switch (payload.tab) {
        case ModelOutputTabs.Bar:
           mutation = PlottingSelectionsMutations.updateBarchartSelections;
            break;
        case ModelOutputTabs.Table:
            mutation = PlottingSelectionsMutations.updateTableSelections;
            break;
        case ModelOutputTabs.Map:
            mutation = PlottingSelectionsMutations.updateOutputChoroplethSelections;
            break;
        case ModelOutputTabs.Bubble:
            mutation = PlottingSelectionsMutations.updateBubblePlotSelections;
            break;
     }
     commit(`plottingSelections/${mutation}`, {payload: payload.payload}, {root: true});       
```

Or could make a dictionary of tab to mutation and select from that - Nick always liked that trick as a way of avoiding switches!